tests/page-objects/search-page.ts

```diff
@@ -61,104 +26,7 @@ export class SearchPage {
-    await this.btnClearInput.click();
-  }
-
-  async clickSearchInputOption(option: SearchOption) {
-    await expect(this.btnCollectionSearchInputGo).toBeVisible({
-      timeout: 5000,
-    });
-    await expect(this.formInputSearchPage).toBeVisible({ timeout: 60000 });
-
-    await this.formInputSearchPage.click({ force: true });
-    await this.page.waitForLoadState('networkidle', { timeout: 60000 });
-    await expect(
-      this.btnCollectionSearchInputCollapser.getByText(option),
-    ).toBeVisible({ timeout: 60000 });
-    await this.btnCollectionSearchInputCollapser
-      .getByText(option)
-      .click({ force: true });
-  }
-
   async goBackToSearchPage() {
     await this.visit();
   }
-
-  async assertClearAllFiltersNotVisible() {
-    await this.page.waitForLoadState();
-    await expect(this.btnClearAllFilters).not.toBeVisible({ timeout: 60000 });
-  }
-
-  async validateTVPage(query: string) {
-    await this.page.waitForLoadState('networkidle', { timeout: 60000 });
-    expect(await this.page.title()).toContain('Internet Archive TV NEWS');
-    await expect(
-      this.page.getByRole('link', { name: 'TV News Archive', exact: true }),
-    ).toBeVisible({ timeout: 60000 });
-    await expect(
-      this.page.getByRole('heading', { name: 'Search' }),
-    ).toBeVisible({ timeout: 60000 });
-    await expect(this.formInputTVPage).toBeVisible({ timeout: 60000 });
-    expect(await this.formInputTVPage.inputValue()).toContain(query);
-  }
-
-  async validateRadioPage(query: string) {
-    await this.page.waitForLoadState('networkidle', { timeout: 60000 });
-    await expect(this.formInputRadioPage).toBeVisible({ timeout: 60000 });
-    expect(await this.formInputRadioPage.inputValue()).toContain(query);
-  }
-
-  async validateWaybackPage(query: string) {
-    await this.page.waitForLoadState('networkidle', { timeout: 60000 });
-    expect(await this.page.title()).toContain('Wayback Machine');
-    await expect(this.formInputWaybackPage).toBeVisible({ timeout: 60000 });
-    expect(await this.formInputWaybackPage.inputValue()).toContain(query);
-  }
-
-  async validateCompactViewModeListLineDateHeaders(filter: SortFilter) {
-    const checkFilterText = filter
-      .split('Date ')[1]
-      .replace(/^./, (str: string) => str.toUpperCase());
-    expect(
-      await this.page
-        .locator('tile-list-compact-header #list-line-header #date')
-        .innerText(),
-    ).toContain(checkFilterText);
-  }
-
-  async validateURLParamsWithSortFilter(filter: SortFilter, order: SortOrder) {
-    const sortFilterURL =
-      order === 'descending'
-        ? `-${SortFilterURL[filter]}`
-        : SortFilterURL[filter];
-    const urlPatternCheck = new RegExp(`sort=${sortFilterURL}`);
-    await expect(this.page).toHaveURL(urlPatternCheck);
-  }
-
-  async validateClearSearchInput() {
-    await expect(this.btnClearInput).not.toBeVisible({ timeout: 60000 });
-    expect(await this.formInputSearchPage.inputValue()).toBe('');
-  }
```

Comment:
q: where did all the search tests go?

Reply:
refactored to be in `pageObjects/collection-browser.ts`